util/generate_build_files.py

```diff
@@ -41,590 +41,7 @@
 
 # NON_PERL_FILES enumerates assembly files that are not processed by the
```

Comment:
Should this comment, then the declared empty list move to the end of the file where the files are actually listed?

Reply:
Updated.